### ISSUE TITLE
Add a few more cases for finding the correct Unreal exe

### ISSUE THREAD

**Issue by Raicuparta** [Raicuparta/rai-pal#154]
Returnal and Sackboy have a Win64/Launcher.exe next to the Win64/*-Shipping.exe.

Jedi Fallen Order somehow has two exes that look identical?
```
SwGame/Binaries/Win64/starwarsjedifallenorder.exe | exe | 58.41 MiB
SwGame/Binaries/Win64/SwGame-Win64-Shipping.exe | exe | 58.41 MiB
```

Current rule just picks the first Win64/*.exe it finds, so no good.

**Comment by Raicuparta** [comment]
Might be fixed in 0.6.1?

**Comment by Raicuparta** [comment]
Gonna close this for now, presumed to be fixed
